documentation-site/components/yard/config/form-control.ts

```diff
@@ -0,0 +1,69 @@
+import {FormControl} from 'baseui/form-control';
+import {Input} from 'baseui/input';
+import {PropTypes} from '../const';
+import {TConfig} from '../types';
+
+const TextareaConfig: TConfig = {
+  imports: {
+    'baseui/form-control': {named: ['FormControl']},
+  },
+  scope: {
+    FormControl,
+    Input,
+  },
+  theme: [],
+  props: {
+    children: {
+      value: '<Input />',
+      type: PropTypes.ReactNode,
+      description: 'Content wrapped by FormControl.',
+      imports: {'baseui/input': {named: ['Input']}},
+    },
+    label: {
+      value: 'label',
+      type: PropTypes.String,
```

Comment:
`label` and `caption` are `ReactNode | Function` type. I think we can go with a `Function` type for the yard case and maybe in the description mention that both types are accepted.

Reply:
Done. We might add some hybrid multi-type to yard in the future. For now, function is the most powerful type, so let's use that.